homeworks/zlinszkygyorgy/7_extract_transform/coingecko.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import settings as s
+import pandas as pd
+
+url = "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&per_page=250&page=1"
```

Comment:
a vs_currency, a per_page  és a page is paraméterek, ezeket is a params dicitonary-ben add hozzá kérlek.
A url csak simán: `https://api.coingecko.com/api/v3/coins/markets`
Ez azért fontos, mert ha több api hívásod lenne, eltérő paraméterekkel, akkor megnehezítenéd a dolgod hogy hard code-ovla vannak paraméterek az url-be.